lib/src/signed_video_h26x_auth.c

```diff
@@ -118,16 +102,23 @@ decode_sei_data(signed_video_t *self, const uint8_t *payload, size_t payload_siz
  * Detects if there are any missing SEI messages based on the GOP counter and updates the GOP state.
  */
 static void
-detect_sei_loss_and_sync_gop(signed_video_t *self)
+detect_lost_sei_and_update_gop_state(signed_video_t *self)
```

Comment:
The name is still a little bit misleading. There is only one gop_state that is updated and there is a gop_state_update(...) function as well...